INSTALL.md

```diff
@@ -103,7 +103,7 @@ $ export ENABLE_STATS_TAGS=true
 $ export ENABLE_STATSD=true
 ```
 
-(Optional) To deploy [statsd_exporter](https://github.com/prometheus/statsd_exporter) as a sidecar, which can recieve statsd metrics and republish them in Prometheus format. It listens for metrics on 127.0.0.1:8125, and exposes a prometheus endpoint at 127.0.0.1:9201. This is useful as statsd provides metrics around request latency (p50, p99 etc), whereas the standard Envoy prometheus endpoint does not.
+(Optional) To deploy [statsd_exporter](https://github.com/prometheus/statsd_exporter) as a sidecar, which can receive statsd metrics and republish them in Prometheus format. It listens for metrics on 127.0.0.1:8125, and exposes a prometheus endpoint at 127.0.0.1:9201. This is useful as statsd provides metrics around request latency (p50, p99 etc), whereas the standard Envoy prometheus endpoint does not.
```

Comment:
This is no longer valid because the statsd exported has been deleted see https://github.com/aws/aws-app-mesh-inject/issues/82

Reply:
@krlevkirill I think Stefan is suggesting that you should remove the above section on the `statsd_exporter` configuration and the corresponding section below.